src/plugins/summaryWidget/test/ConditionManagerSpec.js

```diff
@@ -343,13 +343,14 @@ define(['../src/ConditionManager'], function (ConditionManager) {
 
             expect(telemetryCallbackSpy).not.toHaveBeenCalled();
 
+            var callbackPromise = new Promise(function (resolve){
+                telemetryCallbackSpy.and.callFake(resolve);
```

Comment:
Another spot where `done` would be a better fit?  Use callFake to verify expectations and then call done.  Instead of wrapping in a promise.